src/implementations/vanilla/utilities/clearfix/README.md

```diff
@@ -0,0 +1,3 @@
+# ECL Utility Clearfix
```

Comment:
missing website documentation

Reply:
For this one and the others, I've taken these pages https://github.com/ec-europa/europa-component-library/tree/v2-dev/src/website/src/pages/ec/utilities

Do you mean there is other existing information I've missed?

Reply:
no, the documentation pages do not exist yet, but at some point that would be a good idea to create them.
That's why I suggested to have another ticket to add them (we are the one writing documentation for utilities, as it is not something managed by COMM)

Reply:
FRONT-1786 created as a follow-up
The rest of the feedback points have been covered